components/HomePageComponents/NewLetter.tsx

```diff
@@ -17,7 +17,7 @@ function Newsletter({ translate }: Props): JSX.Element {
         >
           <div>
             <div className="mb-6">
-              <h3 className="text-3xl text-black font-semibold">
+              <h3 className="ztext-3xl text-black font-semibold">
```

Comment:
please change to the correct class name 'text-'

Reply:
since we already have a Newsletter h3 class in the styles.module of this component, no need for the text-black and font-semibold classes as the custome style already accounts for it